website/api/zkvm/tutorials/io.md

```diff
@@ -280,29 +280,29 @@ Happy coding!
-[example-jwt-validator-guest]: https://github.com/risc0/risc0/blob/main/examples/jwt-validator/methods/guest/src/main.rs
-[example-jwt-validator-host]: https://github.com/risc0/risc0/blob/main/examples/jwt-validator/src/lib.rs
-[example-jwt-validator]: https://github.com/risc0/risc0/blob/main/examples/jwt-validator/README.md
-[example-voting-machine-core]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/core/src/lib.rs
-[example-voting-machine-guest-elf-init-commit]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/methods/guest/src/bin/init.rs#L29-L33
-[example-voting-machine-guest-elf-init-read]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/methods/guest/src/bin/init.rs#L28
-[example-voting-machine-guest-elf-init]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/methods/guest/src/bin/init.rs
-[example-voting-machine-guest-elf-submit-commit]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/methods/guest/src/bin/submit.rs#L31-L39
-[example-voting-machine-guest-elf-submit-write]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/methods/guest/src/bin/submit.rs#L30
-[example-voting-machine-host-freeze]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/src/lib.rs#L99-L112
-[example-voting-machine-host-init-write]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/src/lib.rs#L79
-[example-voting-machine-host-init]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/src/lib.rs#L77-L83
-[example-voting-machine-host-submit-output-buffer]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/src/lib.rs#L88
-[example-voting-machine-host-submit-output-from_slice]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/src/lib.rs#L95
-[example-voting-machine-host-submit-output-stdout]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/src/lib.rs#L91
-[example-voting-machine-host-submit]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/src/lib.rs#L85-L97
-[example-voting-machine-host]: https://github.com/risc0/risc0/blob/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine/src/lib.rs
-[example-voting-machine]: https://github.com/risc0/risc0/tree/2deb83a15d8bd438bdc594a86438fc816d02b480/examples/voting-machine
+[example-chess-core]: https://github.com/risc0/risc0/blob/release-0.21/examples/chess/core/src/lib.rs
```

Comment:
We actually want to point this to `main`

Reply:
Thanks!